### ISSUE TITLE
CreateDelta is not handling FULLDL correctly

### ISSUE THREAD

**Issue by matthewrsj** [clearlinux/mixer-tools#205]
CreateDelta attempts to handle FULLDL from bsdiff gracefully and print a message that the delta will not be used. Instead it prints:
```
  Creating delta pack for bundle R-extras from 20510 to 20540
    WARNING: Failed to create delta for /usr/lib64/R/library/microbenchmark/Meta/package.rds (20450) -> /usr/lib64/R/library/microbenchmark/Meta/package.rds (20540): failed to execute bsdiff /srv/mixer-testing/mix/validate-swupd/update/image/20450/full/usr/lib64/R/library/microbenchmark/Meta/package.rds /srv/mixer-testing/mix/validate-swupd/update/image/20540/full/usr/lib64/R/library/microbenchmark/Meta/package.rds /srv/mixer-testing/mix/validate-swupd/update/www/20540/delta/20450-20540-fada51f2682ee6bfc73b776dd7c8e5db319e7c04bc960a5af79d254ea93de721-665defdf97bcb4ff5a45c43c8595137d0439062e079e6e4f6eb6c9e9c0e5b8c4
STDOUT:
Failed to create delta (1)
failed to execute: exit status 1
```

A loud error before continuing. This should be much quieter since it is not an error case, the delta was just not worth it.

**Comment by cmarcelo** [comment]
The code intention was to do this, but apparently the type matching is not working.

https://github.com/clearlinux/mixer-tools/blob/master/swupd/delta.go#L110

**Comment by matthewrsj** [comment]
Fixed by #299
